Resources/views/custom.css.twig

```diff
@@ -270,7 +281,16 @@ abbr[title]  {
     padding-right: 30px;
 }
 .pumukit_mmobj {
-    border-radius: 0;
+    border-radius: 2;
+    border-style: solid;
+    border-top-style: solid;
+    border-right-style: none;
+    border-bottom-style: solid;
+    border-left-style: solid;
+    border-right-style: solid;
+    
+    border-width: 2px;
+    border-color: #003c5e;
```

Comment:
Why not use `border: 2px solid #003c5e; `?

Also, the color #003c5e can be accessed using "{{default_content_font}}" instead (since the .css is generated by twig anyways.)